docs/contributors/versioning.md

```diff
@@ -0,0 +1,23 @@
+# Versioning
+
+When deciding to version your pull request, consider the list. In addition to what is [typically considered](https://semver.org/#summary) for versioning by semver.org, below are a list of examples of what we consider a `Major, Minor, Patch` change. If you find that your change does not match the list, reach out to us for guidance.
+
+## Major
+
+- Removing/renaming component names, arguments, or slots.
+- Removing dependencies from the `primer_view_components.gemspec` or the `package.json`.
```

Comment:
Hmm, why would removing a dep necessitate a major version bump?

Reply:
I was hypothetically thinking of removing a gem like `view_component` but I don't feel really strongly here if you think we should move to Minor status for that

Reply:
Yeah, I think that's a minor bump sort of thing. IMHO it's bad practice for the host project to use a dep they aren't depending on directly.